src/main/java/homework_2/random_chars_table/ParserInputData.java

```diff
@@ -0,0 +1,32 @@
+package homework_2.random_chars_table;
+
+public class ParserInputData {
+
+    public InputData parserInputRandomData(String input) throws ParserError {
+
+        String[] arrayWithInputData = input.trim().split(" ");
+        try {
+            int row = Integer.parseInt(arrayWithInputData[0]);
+            int column = Integer.parseInt(arrayWithInputData[1]);
+
+            String typeOfParity = arrayWithInputData[2];
```

Comment:
where do you check that you have arrayWithInputData.length > 3 ?
ArrayIndexOutOfBoundsException

Reply:
Yep, fixed